checker/checkIntegration

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+CHECKER=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: checkIntegration FRAMEWORK [FRAMEWORK_ARGS]
+Where FRAMEWORK is one of:
+  HBase           \Run HBase job to check HBase with Alluxio integration.
+  Hive            \Run Hive job to check Hive with Alluxio integration.
+  MapReduce       \Run MapReduce job to check MapReduce with Alluxio integration.
+  Spark           \Run Spark job to check Spark with Alluxio integration.
+
+FRAMEWORK_ARGS will be pass to run corresponding FRAMEWORK job.
+
+-h  display this help."
+
+function run_hbase() {
+  ${LAUNCHER} "${CHECKER}/hbase-integration.sh"
+}
+
+function run_hive() {
+  ${LAUNCHER} "${CHECKER}/hive-integration.sh"
+}
+
+function run_mapreduce() {
+  ${LAUNCHER} "${CHECKER}/mapreduce-integration.sh"
+}
+
+function run_spark() {
+   ${LAUNCHER} "${CHECKER}/spark-integration.sh" "$1"
+}
+
+function main {
+  case "$1" in
+    HBASE | HBase | hbase)
```

Comment:
Other than the enumeration, it's better to define a util function to lower the case and then compare.

https://stackoverflow.com/questions/2264428/converting-string-to-lower-case-in-bash